pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   func: ">=0.1.0 <2.0.0"
   http: '^0.11.3'
   js: "^0.6.0"
+  service_worker: ^0.1.2
```

Comment:
This is only used in the example, right? Should be in `dev_dependencies`

Reply:
yep will move it